python/cudf/cudf/tests/test_binops.py

```diff
@@ -88,7 +88,16 @@ def test_series_binop_scalar(nelem, binop, obj_class):
 _bitwise_binops = [operator.and_, operator.or_, operator.xor]
 
 
-_int_types = ["int8", "int16", "int32", "int64", "longlong"]
+_int_types = [
```

Comment:
So, in general, it looks like a lot of tests need a list of all numeric types (or just signed/unsigned types). Instead of repeating everywhere, let's have a `tests.utils.SIGNED_TYPES`, `tests.utils.UNSIGNED_TYPES`, `tests.utils.NUMERIC_TYPES`, etc.?

Reply:
Or maybe `cudf.utils.dtypes.*TYPES`.

Reply:
That works, it would be better.

Reply:
we could get clever do something like `cudf.utils.dtypes.ALL_TYPES - ("str", float64")`. :imp: